### PR TITLE
support for Mongoid 7

### PR DESCRIPTION
**Pull request by tomasc**

### CODE REVIEW

**Comment by tomasc** [comment]
@frodsan @gr4y @chancancode @prasadsurase @rmm5t can we get this merged please and have a new gem out? Would be fantastic to be able to use `mongoid-minitest` with the new Mongoid 7. Thank you!

File: mongoid-minitest.gemspec [hunk 1]
```diff
@@ -21,8 +21,9 @@ Gem::Specification.new do |gem|
   gem.required_ruby_version = '>= 1.9.3'
 
   gem.add_dependency 'minitest', '~> 5.0'
-  gem.add_dependency 'mongoid' , '>= 3'
+  gem.add_dependency 'mongoid' , '>= 3', '<= 7.0'
```
Comment (rmm5t) [inline mongoid-minitest.gemspec]: Shouldn't this be `'>=3', '< 8.0'`?

**Review by rmm5t — CHANGES_REQUESTED** [review]
Overall looks good upon a quick pass. Just one minor change to the gemspec, and I think we can get this merged in.

**Review by tomasc — COMMENTED** [review]
Fine with me, will amend

**Comment by tomasc** [comment]
@rmm5t updated, ready for a merge. Thank you.

**Review by rmm5t — APPROVED** [review]

**Comment by rmm5t** [comment]
[v1.2.0 just released](https://rubygems.org/gems/mongoid-minitest)

Thanks for the help! 🍻

**Comment by tomasc** [comment]
Great, thank you!

**Pull request merged**